### DISCUSSION TITLE
newbie in ros2 and rclnodejs help

### DISCUSSION THREAD

**Discussion by AceOfDema** [RobotWebTools/rclnodejs#961] [Q&A] [answered]
hi im here because i dont know where to look, im trying to include a custome package which the varius msg are defined and i want to import that to my index.js file
here my directory structure
-src
--hmi
---include
---launch
---src
----index.js
---{all the generated files using rclnodejs-cli}
--interfaces
---srv
---msg
----test.msg

i have no idea how to make include this ros2 package inside index.js, i added the find_package in cmakelist.txt, package.js

**Comment by minggangw** [comment, marked as answer]
What you need to do is **source** your project's local_setup.bash, like:
```bash
source /<path_to_your_project>/local_setup.bash
```
